cloud/shared/bin/deploy.py

```diff
@@ -0,0 +1,16 @@
+import subprocess
+import importlib
+import os
+
+
+def run(config):
```

Comment:
Can you add type hint here:

```
def run(config: ConfigLoader): 
   ...
```

This `run` function is not directly called anywhere so it might be challenging to figure out what `config` here is. 

Same for all other `run` functions in this PR.